src/utils/helpers/Selectors.ts

```diff
@@ -49,8 +49,16 @@ export function selectDetailsZoom() {
     return selectById<SVGGElement>(ElementIds.ZOOM_DETAILS);
 }
 
-export function selectContainer() {
-    return selectById<SVGSVGElement>(ElementIds.CONTAINER);
+export function selectOverviewZoom() {
```

Comment:
Can you change to ZoomOverview?

Reply:
Overview is a name I chose for main(force-graph) view. In this context OverviewZoom is a zoom related to Overview screen and I think it is correctly named. 

I can change name of Overview to something like MainGraph or MainView if the current name isn't intuitive enough

Reply:
So maybe change id to OVERVIEW_ZOOM?